src/openloco/company.h

```diff
@@ -17,9 +17,9 @@ namespace openloco
 
     enum company_flags
     {
-        challenge_completed = (1 << 6),
-        challenge_failed = (1 << 7),
-        challenge_flag8 = (1 << 8),
+        challenge_completed = (1 << 6), // 0x40
+        challenge_failed = (1 << 7),    // 0x80
+        challenge_enabled = (1 << 8),   // 0x100
```

Comment:
I maintain that `challenge_enabled` is not a good description for this flag. It appears to be the opposite, as it triggers `string_id::empty` in the challenge string.